### PR TITLE
Add location page, update questions

### LINKED ISSUE #127: [FEATURE] Update questions to final reviewed list
Questions are here:
https://covidnearme.org/api/v1/checkup_questions

They may be hardcoded, since we don't expect them to change per Elaine.

### PR DESCRIPTION
**Pull request by gspencergoog**
This adds a new location page that is part of the questions asked.

It switches between a location in the US, and a location in another country, validating inputs as much as it can.

As part of the update, I also modified all the questions to match the "canonical" list of questions in the doc, and eliminated `SimpleSlider`, since it's not used anymore.

Fixes #127

Here's what it looks like:

image

image

image

### CODE REVIEW

File: lib/src/data/models/checkups.dart [hunk 1]
```diff
@@ -77,15 +77,18 @@ class VitalsResponse {
 @JsonSerializable()
 class CheckupLocation {
   String postalCode;
```
Comment (joshua-s) [inline lib/src/data/models/checkups.dart]: Since zip code is being used elsewhere, we could probably rename this too.

Comment (ferndot) [inline lib/src/data/models/checkups.dart]: Could you rename this to "zip code" for consistency with the rest of the app? The backend can be updated to match.

File: lib/src/data/models/checkups.dart [hunk 2]
```diff
@@ -76,16 +76,19 @@ class VitalsResponse {
...
   });
 
   factory CheckupLocation.fromJson(Map<String, dynamic> json) =>
       _$CheckupLocationFromJson(json);
   Map<String, dynamic> toJson() => _$CheckupLocationToJson(this);
 
   @override
-  String toString() => 'CheckupLocation { postalCode: $postalCode }';
+  String toString() =>
+      'CheckupLocation { postalCode: $zipCode, country: $country }';
```
Comment (ferndot) [inline lib/src/data/models/checkups.dart]:
```suggestion
      'CheckupLocation { zipCode: $zipCode, country: $country }';
```

**Comment by ferndot** [comment]
Eventually, it would be nice if the country input could be a searchable dropdown. Each item would be labeled with the country's full name and use the corresponding ISO 3166-1 alpha-2 code as its value.

Since this PR is already quite large, we could handle that in a follow-up PR if you'd prefer not to expand this further.

**Comment by gspencergoog** [comment]
Yes, I agree it would give more reliable data to have a pulldown. I'll do that in another PR as soon as this is landed.

File: lib/src/data/models/checkups.dart (hunk 2 above)
Comment (gspencergoog) [reply lib/src/data/models/checkups.dart]: Fixed.

**Review by ferndot — CHANGES_REQUESTED** [review]
This is great 👍 ! There is one unused import that is breaking the tests. Once that is fixed, I think this is ready to go:
```
info • Unused import: 'package:covidnearme/src/ui/widgets/questions/inputs/entry_field.dart' • lib/src/ui/screens/checkup/steps/temperature.dart:1:8 • unused_import
```

One callout: some of the questions from the MVP questionnaire are not included. However, these can easily be added in a followup PR. I've added a list of these below for clarity.

Missing questions/questions that need changes:
- How high did your temperature get today?
- Have you been tested for flu, pneumonia, or other respiratory illness?
- Was the test positive?
- What did you have?
- Did you try to get tested for COVID-19?
- What was the result?
- Shortness of breath question should not have a subtitle.

**Comment by gspencergoog** [comment]
OK, I fixed the extra import.  I'll add the other questions in another PR.

**Review by ferndot — APPROVED** [review]
LGTM!

**Comment by goderbauer** [comment]
One issue I noticed when playing around with this PR: The keyboard doesn't get dismissed when you hit the next button after entering a ZIP code.

**Pull request merged**